floats_test.go

```diff
@@ -1444,3 +1444,15 @@ func BenchmarkAddScaledToLarge(b *testing.B) {
 func BenchmarkAddScaledToHuge(b *testing.B) {
 	benchmarkAddScaledTo(b, Huge)
 }
+
+func benchmarkScale(b *testing.B, size int) {
+	dst := randomSlice(size)
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		Scale(2.0, dst)
```

Comment:
Does this run into issues because all of the numbers are going to overflow? Would making it 1.01 or something be better?